sdc/hiframes/hiframes_untyped.py

```diff
@@ -759,7 +759,8 @@ def get_dtypes(df):
     @staticmethod
     def infer_column_names_and_types_from_constant_filename(fname_const, skiprows, col_names, sep=','):
```

Comment:
Does it used somewhere?

Reply:
It is part of the old code. I think it is possible to clean it in new PR.

Reply:
I will remove it from PR